pyiron_atomistics/atomistics/structure/periodic_table.py

```diff
@@ -5,10 +5,10 @@
 from __future__ import print_function, unicode_literals
 import numpy as np
 import os
-from pyiron_base import state
 import mendeleev
 import pandas
 from functools import lru_cache
+import pyiron_atomistics
```

Comment:
Can you use the `__file__` from the `periodic_table.py` and define the path relative to this one?

Reply:
A bit more fragile but a-ok for me. You want to minimize the number of imports? Will do it tomorrow :)